### PR TITLE
improve indentation of step-18 and minor code cleanup

### PR DESCRIPTION
**Pull request by davydden**
previous indentation https://github.com/dealii/dealii/blob/dealii-9.0/examples/step-18/step-18.cc#L1020-L1023

### CODE REVIEW

File: examples/step-18/step-18.cc [hunk 1]
```diff
@@ -993,8 +987,11 @@ namespace Step18
                     eps_phi_i = get_strain(fe_values, i, q_point),
                     eps_phi_j = get_strain(fe_values, j, q_point);
 
-                  cell_matrix(i, j) += (eps_phi_i * stress_strain_tensor *
-                                        eps_phi_j * fe_values.JxW(q_point));
+                  cell_matrix(i, j) += (eps_phi_i *             //
+                                        stress_strain_tensor *  //
+                                        eps_phi_j               //
+                                       ) *                      //
+                                       fe_values.JxW(q_point)); //
```
Comment (drwells) [inline examples/step-18/step-18.cc]: The tester says that this is an extra parenthesis.

Comment (davydden) [reply examples/step-18/step-18.cc]: ha, true. Will fix.

File: examples/step-18/step-18.cc [hunk 2]
```diff
@@ -1826,42 +1810,13 @@ namespace Step18
...
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Unknown exception!" << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      return 1;
-    }
+  TopLevel<3> elastic_problem;
+  elastic_problem.run();
```
Comment (drwells) [inline examples/step-18/step-18.cc]: From #6784: we should keep the handler so that MSVC can print error messages.

Comment (davydden) [reply examples/step-18/step-18.cc]: @drwells done.

**Pull request merged**